core/src/test/resources/sql/sub-query.iq

```diff
@@ -63,6 +63,20 @@ select * from dept where deptno not in (select deptno from emp);
 (0 rows)
 
 !ok
+
+# [CALCITE-5638] Assertion Failure during planning correlated query
```

Comment:
split query over multiple lines, one clause per line, for readability.

Reply:
ok, done